### PR TITLE
Ninjachat GTM code

### PR DESCRIPTION
**Pull request by nagendra-500**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
-# 500apps-public-gtm-ninjachat
\ No newline at end of file
+# Ninjachat
+[Get Started](https://infinity.500apps.com/ninjachat) with with [Ninjachat](https://ninjachat.com)
+
+A live chat application with complex features but with a high ease of use quotient. Ninjachat has all the features one needs to increase and improve customer engagement. Personalise the chat widgets, connect with visitors and respond to them in real time. Ninjachat helps boost customer retention with no glitches thereby providing a good user experience.
```
Comment (agilecrm) [inline README.md]: This needs changes

Comment (nagendra-500) [reply README.md]: Modified Sir.

**Comment by subramaniyamkolluri** [comment]
video

**Comment by nagendra-500** [comment]
> video

@subramaniyamkolluri Sir,Please find below video 
https://watch.screencastify.com/v/k30B9aYP1xBLnY5KIWAm

File: README.md [hunk 2]
```diff
@@ -1 +1,42 @@
-# 500apps-public-gtm-ninjachat
\ No newline at end of file
+# Ninjachat
+[Get Started](https://infinity.500apps.com/ninjachat) with with [Ninjachat](https://ninjachat.com)
```
Comment (subramaniyamkolluri) [inline README.md]: why two with

File: README.md [hunk 3]
```diff
@@ -1 +1,42 @@
...
+
+A live chat application with complex features and also with a high ease of use quotient. Ninjachat has all the features one needs to increase and improve customer engagement. Personalise the chat widgets, connect with visitors and respond to them in real time. Ninjachat helps boost customer retention with no glitches thereby providing a good user experience.
+Create departments and install the script generated in your websites to run the Ninjachat widget. Generate leads, resolve customer complaints, address issues, etc with easy-to-use Ninjachat application. As the customer traction of your website grows, you can add more agents to the application, where each agent is equipped with handling 25 simultaneous live chats.You can also restrict the number of live chats an agent can do, there by giving time for new agents to get trained before scaling up the operations. Customise user experience tracking live chats and by building custom short cuts as well.
```
Comment (subramaniyamkolluri) [inline README.md]: after period give a space

File: README.md [hunk 4]
```diff
@@ -1 +1,42 @@
...
+- Read through the GTM Terms of Service. Click Yes to accept them and continue.
+
+You will get a container with a code snippet. Copy the code and paste it into every page of your site in the recommended places.
+Click Ok to close the code snippet box and finish setting up the GTM manager and your first container.
+
+Adding the ChatBot tag to your website
+Once you're logged in to your account with the container, it's time to add a new tag with the ChatBot tracking code inside it. To do so, follow these steps   
+In the Google Tag Manager Home, click on the New Tag tile.
+Click on the Tag Configuration tile and select Custom HTML from the tags list.
+Paste the ChatBot tracking code to the HTML field.
```
Comment (subramaniyamkolluri) [inline README.md]: should it be just api key. Why the complete scripting code?

**Comment by subramaniyamkolluri** [comment]
> > video
> 
> @subramaniyamkolluri Sir,Please find below video https://watch.screencastify.com/v/k30B9aYP1xBLnY5KIWAm

The type is wrong for both Botup and Ninjachat

Why name for Botup says Botup Script

1. Please give me suggestion sir.Then I will change type in template.tpl file

2. In my local only botup says botup script (checking purpose I added previously.The generated tpl is not pushed in botup-gtm sir)

File: README.md (hunk 2 above)
Comment (nagendra-500) [reply README.md]: Removed sir

File: README.md (hunk 3 above)
Comment (nagendra-500) [reply README.md]: Given space after period

File: README.md (hunk 4 above)
Comment (nagendra-500) [reply README.md]: @subramaniyamkolluri Sir.Above content is given by @indira. Previoulsy she gaves content for botup also
https://github.com/agilecrm/500apps-public-gtm-botup/pull/1/files

Comment (nagendra-500) [reply README.md]: Modified Sir

File: README.md [hunk 5]
```diff
@@ -1,49 +1,57 @@
...
+
+2. Click on the Tag Configuration tile and search for our community template 'Ninjachatby 500apps (Chatbot Software)' in the 'Community Template Gallery'.
+
+3. Click on ‘Add to workspace’ to select the template and add it to your workspace.
 
-- Sign in to your Google account.
+4. After the template is added to your workspace, the tag is automatically added to your tag configuration.
 
-- Go to the Google Tag Manager home and click on the Create Account button on the right
+5. Find the randomly generated Ninjachat key for the department in the share script slide bar, copy and place the key in the field provided in the Tag Configuration.
```
Comment (subramaniyamkolluri) [inline README.md]: Just say api key not random generated ninjachat key

Comment (nagendra-500) [reply README.md]: Modified Sir

File: README.md [hunk 6]
```diff
@@ -34,7 +34,7 @@ Adding the Ninjachat tag to your website once you're logged in to your account w
 
 4. After the template is added to your workspace, the tag is automatically added to your tag configuration.
 
-5. Find the randomly generated Ninjachat key for the department in the share script slide bar, copy and place the key in the field provided in the Tag Configuration.
+5. Find the API key for the department in the share script slide bar, copy and place the key in the field provided in the Tag Configuration.
```
Comment (subramaniyamkolluri) [inline README.md]: its side bar not slide bar. Actually its given better GTM Insutctions

Comment (nagendra-500) [reply README.md]: Changed Sir

File: README.md [hunk 7]
```diff
@@ -34,7 +34,7 @@ Adding the Ninjachat tag to your website once you're logged in to your account w
 
 4. After the template is added to your workspace, the tag is automatically added to your tag configuration.
 
-5. Find the API key for the department in the share script slide bar, copy and place the key in the field provided in the Tag Configuration.
+5. Find the API key for the department in the share script side bar, copy and place the key in the field provided in the Tag Configuration.
```
Comment (subramaniyamkolluri) [inline README.md]: Do we have GTM Instructions similar to PushNinja in the app

Comment (nagendra-500) [reply README.md]: Yes Sir.

Comment (subramaniyamkolluri) [reply README.md]: Can you show me the share script side bar

Comment (nagendra-500) [reply README.md]: @subramaniyamkolluri Sir
 share script sidebar in app: https://snipboard.io/IOnfq2.jpg
 Google tag manager:https://snipboard.io/ot6CfR.jpg

**Comment by subramaniyamkolluri** [comment]
User will not know where and which one to copy


We may have to show gtm icon like pushninja and include those details here

**Comment by nagendra-500** [comment]
> User will not know where and which one to copy
> 
> We may have to show gtm icon like pushninja and include those details here

Added Sir: #https://github.com/agilecrm/500appsInfinity/pull/7178

**Comment by subramaniyamkolluri** [comment]
Manohar. Pl review. Unable to tag you in mobile view for this repo

**Comment by sumit-500** [comment]
> Manohar. Pl review. Unable to tag you in mobile view for this repo

@agilecrm  Sir, Your attention Please

File: README.md [hunk 8]
```diff
@@ -1 +1,57 @@
-# 500apps-public-gtm-ninjachat
\ No newline at end of file
+# Ninjachat by 500apps
```
Comment (agilecrm) [inline README.md]: can we say 500apps Ninjachat

So that it comes as the first app in the GTM

File: template.tpl [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+
+
+___INFO___
+
+{
+  "type": "TAG",
+  "id": "cvt_temp_public_id",
+  "version": 1,
+  "securityGroups": [],
+  "displayName": "Ninjachat by 500apps",
```
Comment (agilecrm) [inline template.tpl]: 500apps NinjaChat

Comment (nagendra-500) [reply template.tpl]: Modified Sir

File: README.md (hunk 8 above)
Comment (nagendra-500) [reply README.md]: Modified Sir

**Pull request merged**